docs/content/tutorials/creating-a-new-theme.md

```diff
@@ -416,9 +416,23 @@ When rendering the site, Hugo will create new files and update existing ones in
 
 As of version 0.15, Hugo doesn't write files when running in server mode. Instead, it keeps all the rendered files in memory. You can "clean" up files by stopping and restarting the server.
 
-### Hugo's Watch Option
+### Watching with Hugo
 
-Hugo's "`--watch`" option will monitor the content/ and your theme directories for changes and rebuild the site automatically. Since version 0.15, this has been the default option for `hugo server`.
+Hugo's watch functionality monitors the relevant site and theme directories
+for file changes,
+and rebuilds the site automatically when changes are detected.
+
+By default, watching is
+enabled when in web server mode (`hugo server`),
+but disabled for the web site generator (`hugo`).
+
+In some use cases,
+the Hugo web site generator should watch&mdash;simply
+type `hugo --watch` on the command line.
+
+To deploy to slugs with read-only filesystems,
+the Hugo web server obviously shouldn't watch&mdash;so
+use `hugo server --watch=false`.
```

Comment:
I don't like the use of "slugs" here.  That sentence will only make sense to people familiar with Heroku (esp. since "slugs" has a different meaning inside of Hugo).  Say something like this:

```
In some use cases such as Docker containers and Heroku slugs, the site sources may live on a read-only filesystem.
It doesn't make sense to watch for file changes in that scenario, so use `hugo server --watch=false`.
```